components/calendar/nz-calendar-header.component.ts

```diff
@@ -1,8 +1,10 @@
-import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, OnInit, Output, ViewEncapsulation } from '@angular/core';
 import setMonth from 'date-fns/set_month';
 import { NzI18nService as I18n } from '../i18n/nz-i18n.service';
 
 @Component({
+  encapsulation: ViewEncapsulation.None,
```

Comment:
This is the default option used when not having any style: https://github.com/angular/angular/blob/d83307adab10ab4477715bc904401fb78bac17e9/packages/compiler/src/directive_normalizer.ts#L186-L189

But should be fine if made consistent among other components.